tools/debug/dat.py

```diff
@@ -0,0 +1,198 @@
+from time import time
```

Comment:
1. Need Samsung license
2. Could you let me know how to use and what result is? Need README.md to introduce DAT.